src/Microsoft.PowerPlatform.Test.AppWriter/Program.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.PowerPlatform.PowerApps.Persistence.MsApp;
+using Microsoft.PowerPlatform.PowerApps.Persistence.Models;
+//using Microsoft.PowerPlatform.PowerApps.Persistence.Yaml;
+using Microsoft.PowerPlatform.PowerApps.Persistence.Templates;
+using Microsoft.PowerPlatform.PowerApps.Persistence.Extensions;
+
+namespace Test.AppWriter;
+
+internal class Program
+{
+    private static ServiceProvider ConfigureServiceProvider()
+    {
+        var serviceCollection = new ServiceCollection();
+        serviceCollection.AddPowerAppsPersistence(true);
+        var serviceProvider = serviceCollection.BuildServiceProvider();
+        return serviceProvider;
+    }
+
+    private static App GetExampleApp(ServiceProvider provider, string appname, int numscreens = 1)
+    {
+        var app = provider.GetRequiredService<IControlFactory>().CreateApp(appname);
+        for (var i = 0; i < numscreens; i++)
+        {
+            var graph = provider.GetRequiredService<IControlFactory>().CreateScreen("Screen" + i.ToString(),
+                properties: new Dictionary<string, ControlPropertyValue>()
+                {
+                    { "Text", new() { Value = "I am a screen" }  },
```

Comment:
I assume "text" property will become a parameter

Reply:
Yes, the values currently hardcoded within GetExampleApp will be parameterized and controlled via the user interface (command line or GUI) in the future.